TestAppTypescript/package.json

```diff
@@ -7,30 +7,30 @@
     "tsc": "tsc",
     "watch": "tsc -w",
     "ios": "npm run build && concurrently -r \"npm run watch\" \"react-native run-ios\"",
-    "android": "npm run build && concurrently -r \"start npm run watch\" \"start react-native run-android\"",
+    "android": "npm run build && concurrently -r \"npm run watch\" \"react-native run-android\"",
```

Comment:
Question - do we still need `start` in this command?

Reply:
@dhei I don't think so. At least, ios command doesn't have this `start` so I've removed it from `android` one.

Reply:
Also, `npm run ios` was working and `npm run android` wasn't